src/index.js

```diff
@@ -18,7 +18,7 @@ class Origin {
     ipfsGatewayPort = process.env.IPFS_GATEWAY_PORT,
     ipfsGatewayProtocol = process.env.IPFS_GATEWAY_PROTOCOL,
     attestationServerUrl = `${process.env.BRIDGE_SERVER}/api/attestations`,
-    discoveryServer = process.env.DISCOVERY_SERVER_PORT,
+    discoveryServer = process.env.DISCOVERY_SERVER_URL,
```

Comment:
If we are using URL, do we still need discoveryServerPort ? Perhaps let's get rid of it  to make things simpler. So then for local development discoveryServer would point to something like "http://localhost:4000"

Reply:
Yup agree no need for 2 properties